src/react-native/11-navigation/navigation.md

```diff
@@ -2,23 +2,120 @@
 @parent learn-react-native 11
 @outline 3
 
-@description TODO
+@description Learn how to use navigation to move between screens in a React Native app.
 
 @body
 
 ## Overview
 
 In this section, you will:
 
-- TODO
+- Understand why routing is important
+- Be introduced to deep linking
+- Learn about stack-based routing
+- Understand the navigation lifecycle
+- Implement a bottom tab navigation pattern
+- Learn about the difference between stack and native stack navigation
+- Implement a stack navigation pattern
 
 ## Objective 1: Using React Native bottom tab navigation
 
-TODO
+As our application grows in complexity, more screens will be added. The React Navigation library provides a solution that allows us to move between these screens. In this section, we will cover both common navigation patterns used in both Android and iOS applications: tab navigation and stack navigation.
 
-### Concept TODO
+The first step is to install the React Navigation library. Run the following command in your terminal:
 
-TODO
+```bash
+npm install @react-navigation/native
+```
+
+### What is routing?
+
+Routing in React Native refers to the process of managing navigation between different screens in our application. It allows users to move between different parts of our application by clicking on links, buttons, or other interactive elements.
```

Comment:
“Routing” seems like a very web-oriented term. The Academy content is for a web dev audience, but also for folks who might not be familiar with web terms. The Cayman Islands devs have a background in .NET development and are rewriting their app from that to RN.

I think I would update this section to be something like:

### What is navigation?

Navigation in React Native refers to the process of moving between different screens in our application. It helps users transition between various parts of your application through interactions like tapping on buttons, list rows, and other UI elements.

Reply:
updated